pkgs/applications/video/hyperion-ng/default.nix

```diff
@@ -0,0 +1,47 @@
+{ fetchFromGitHub
+, lib
+, cmake
+, mkDerivation
+, libxcb
+, qtbase
+, qtsvg
+, python3
+, qtserialport
+, mbedtls
+, libcec
+, flatbuffers
+, protobuf
+, libusb1
+, hidapi
+, libXrandr
+, qtx11extras
+, libX11
+, avahi-compat
+, wrapQtAppsHook
+}:
+
+mkDerivation rec {
+  pname = "hyperion.ng";
+  version = "2.0.12";
+
+  src = fetchFromGitHub {
+    owner = "hyperion-project";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-J31QaWwGNhIpnZmWN9lZEI6fC0VheY5X8fGchQqtAlQ=";
+  };
+
+  buildInputs = [ libxcb qtbase qtsvg python3 qtserialport mbedtls libcec flatbuffers protobuf libusb1 hidapi libXrandr qtx11extras libX11 avahi-compat wrapQtAppsHook];
```

Comment:
`wrapQtAppsHook` should go to `nativeBuildInputs`

(btw I actually prefer one package per line, also will be nice to format the file with nixpkgs-fmt)